### ISSUE TITLE
regexp for hit Organism

### ISSUE THREAD

**Issue by kyrenya** [tripal/tripal_analysis_blast#28]
Hi, just a little comment
I'm testing the regexp for Hit Organism, and for me 
`^.*?OS=(.*?)\s\w\w=.*$`
doesnt work for detecting Arabidopsis thaliana in my entry: 

<Hit_def>sp|P43288|KSG1_ARATH Shaggy-related protein kinase alpha OS=Arabidopsis 
thaliana GN=ASK1 PE=2 SV=3</Hit_def>

but 
`^.*?OS=(.*?)\s\w=.*$`
works fine  :) 

I post it here just in case its useful for someone not expert in regexp as me :) 
thx bye!

**Comment by spficklin** [comment]
Hi @kyrenya 

I think this is the regular expression you want:   `^.*?OS=(.*?)\s\w\w=.*$`

![image](https://user-images.githubusercontent.com/1719352/49389251-ae289a00-f6da-11e8-95f3-a0f648312d90.png)


I copied that from the online documentation here:  https://tripal.readthedocs.io/en/latest/user_guide/example_genomics/func_annots/blast.html.

**Comment by spficklin** [comment]
Okay, I just realized I got a bit confused there. GitHub munged the regular expression you posted above. I edited so that the stars, `*`, would show up.   So I cut and pasted the regular expression from the online user's guide above which appears to be exactly the same as what you posted above and it works on the regex101.com site.  

I think the one you posted may have a mistake in it.   It needs two `\w` before the `=`

![image](https://user-images.githubusercontent.com/1719352/49389460-3444e080-f6db-11e8-9439-e74e1accf36f.png)
